mobile-browser-based-version/server/models.js

```diff
@@ -0,0 +1,36 @@
+const tf = require('@tensorflow/tfjs')
+
+
+function createTitanicModel() {
```

Comment:
how do we make sure the model architecture is kept compatible with the client-side task code? can the model itself only be defined here and be removed from the client-side code?

Reply:
yes, that is the idea. We will remove the model creation from the front end once this is integrated.